languoids/tree/afro1255/semi1276/west2786/cent2236/sayh1236/qata1238/md.ini

```diff
@@ -13,7 +13,7 @@ links =
 	https://en.wikipedia.org/wiki/Qatabanian_language
 longitude = 45.733333
 latitude = 14.8
-timespan = -0500-01-01/0200-01-01
+timespan = -500-01-01/0200-01-01
```

Comment:
@xflr6 fixed the years in timespans to have 4 digits. Is there a way to keep them as is?